README.md

```diff
@@ -1,2 +1,78 @@
-# rbac-api-utils
-A library for multi cluster rbac api
+# RBAC API Utils
+[![License](https://img.shields.io/:license-apache-blue.svg)](http://www.apache.org/licenses/LICENSE-2.0.html)
+
+A library for multi-cluster RBAC API. Access control in OCM is based on RBAC contructs  provided by Kubernetes,
+this library provides utiilies and  wrappers around the kubernetes RBAC API for convenience and 
+to achieve finer grained access control.
+
+Go to the [Contributing guide](CONTRIBUTING.md) to learn how to get involved.
+
+- The `rbac-api-utils` repository is part of the `open-cluster-management`
```

Comment:
I'm not sure that this is true since it's in Stolostron.

Reply:
I wondered about this but non OCM component Repos like Cert, Iam or Search etc  all seemed to have the same text.